tests/utils_tests.py

```diff
@@ -1316,3 +1316,30 @@ def test_get_form_data_globals(self) -> None:
             )
 
             self.assertEqual(slc, None)
+
+    def test_log_this(self) -> None:
```

Comment:
add a todo to add other test cases that cover log_this logic, could be fun onboarding task for someone :)